examples/open_loop_control_urdfbot/generate.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# generate series of actions for fetch agent on different parts such as moving, arm and gripper.
+
+import numpy as np
+import pandas as pd
+
+wheel_joints = ["wheel_joint_r", "wheel_joint_l"]
+arm_joints = ["arm_joint_0", "arm_joint_1", "arm_joint_2", "arm_joint_3", "arm_joint_4", "arm_joint_5", "arm_joint_6"]
+gripper_joints = ["gripper_finger_joint_r", "gripper_finger_joint_l"]
+other_joints = ["head_pan_joint", "head_tilt_joint"]
+
+arm_poses = np.pi / 180 * np.array([
+    np.array([0, 0, 0, 0, 0., 0, 0.]),
```

Comment:
use 0.0 instead of 0 for consistency with our other python code

Reply:
They are cherry-picked from iGibson:  [arm_default_joint_positions ](https://github.com/StanfordVL/iGibson/blob/783a38ef4962a8757a75c1ece76450163e2c7654/igibson/examples/robots/ik_example.py#L64) and [untucked_default_joint_pos](https://github.com/StanfordVL/iGibson/blob/783a38ef4962a8757a75c1ece76450163e2c7654/igibson/robots/fetch.py#L100)